sdk/core/core/src/az_http_request_builder.c

```diff
@@ -137,14 +136,14 @@ AZ_NODISCARD az_result az_http_request_builder_set_query_parameter(
 
   // Append either '?' or '&'
   p_hrb->url.begin[p_hrb->url.size] = first_parameter ? '?' : '&';
-  ++(p_hrb->url.size);
+  p_hrb->url.size += 1;
```

Comment:
Hey?! That's my artistic handwriting, as an author! :)
Just kidding. If it makes you feel more comfortable - feel free, I understand that sometimes you need to feel more comfortable with code aesthetics, especially that in this case I'm fine either way.
Well, at least I know you probably have read my code :)

Reply:
I think, it come from design guidance. @barcharcraz knows better :-)

Reply:
Tyranny :)